src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -739,7 +739,7 @@ public void TestStartWithMissingFile(bool fullPath)
             Assert.NotEqual(0, e.NativeErrorCode);
         }
 
-        [PlatformSpecific(TestPlatforms.Windows)]
+        [PlatformSpecific(TestPlatforms.Windows)]  // Needs permissions on Unix
```

Comment:
I don't think this comment is accurate.  This test would still fail on Unix even if you ran it as root, no?